aiohttp/client_exceptions.py

```diff
@@ -38,14 +39,34 @@ class ClientResponseError(ClientError):
     """
 
     def __init__(self, request_info, history, *,
-                 code=0, message='', headers=None):
+                 code=None, status=0, message='', headers=None):
         self.request_info = request_info
-        self.code = code
+        self.status = status
+        if code is not None:
+            warnings.warn("code argument is deprecated, use status instead",
+                          DeprecationWarning,
+                          stacklevel=2)
+            if status == 0:
```

Comment:
Please add two a couple lines for raising `ValueError` if both `code` and `status` are provided

Reply:
Done.